server-files/index.js

```diff
@@ -13,23 +14,31 @@ const twitter = new twit({
 app.use(express.static(__dirname + './../public'))
 
 app.get('/tweets/:location', (req, res) => {
-  twitter.get('search/tweets', { q: req.params.location, result_type: 'recent', lang: 'en', count: 50 }, (err, data, response) => {
-    if (err) console.log(err)
-    data.statuses.forEach((element, index) => {
-      const twitterSentiment = sentiment(element.text)
+  knexFunc.incrementCount(req.params.location)
```

Comment:
`knexFunc` is not a very descriptive name. This is `locationSearches` or `searches` or something, right?